dkg/frost/frost.go

```diff
@@ -0,0 +1,756 @@
+package frost
+
+import (
+	"crypto/rand"
+	"encoding/json"
+	"fmt"
+
+	"github.com/bloxapp/ssv-spec/dkg"
+	"github.com/bloxapp/ssv-spec/types"
+	"github.com/coinbase/kryptology/pkg/core/curves"
+	"github.com/coinbase/kryptology/pkg/dkg/frost"
+	"github.com/coinbase/kryptology/pkg/sharing"
+	ecies "github.com/ecies/go/v2"
```

Comment:
is this a well known and well used lib? I saw go-eth has an implementation as well